41.md

```diff
@@ -0,0 +1,46 @@
+NIP-41
+======
+
+### Poll & Vote Event
+-----------------------------------
+
+`draft` `optional` `author:0xtlt`
+
+An event with kind `1` that wants to contain a poll can do so by adding poll tags and its content as follows:
+
+```json
+{
+  ...
+  "kind": 1,
+  "tags": [
+    ...
+    ["poll", "Choice 1"],
+    ["poll", "Choice 2"],
+    ["poll", "Choice 3"]
+    ...
+  ],
+}
+```
+
+Then the client should display voting options if more than **1 choice** is present in the poll.
```

Comment:
Only one `"poll"` may seem pointless, but perhaps it'd be useful for an "Agree", "No", "Sign Up", "ACK" or whatever where ignoring it is an implicit no? So, I think it'd make sense to still show it if only one choice is present, even if that seems odd/niche

Reply:
Poll and sign up for something are two differents things, And we need an explicite usage of the poll, then if only one poll is defined, we can't decide for the user if he wanted to have "agree", "no" or sign up"